### ISSUE TITLE
Build fails under Fedora 21 using Qt4

### ISSUE THREAD

**Issue by DrOptix** [eteran/edb-debugger#139]
When I tried to build the project I got some errors about `edisassm`. Please check this pastebin link for the full compilation log.

http://pastebin.com/j31MxQmu

I also could not be able to use `git submodules` when I cloned `edb-debugger`, because of this I symlinked `src/edisassm` and `src/qhexview` to their stand alone repositories.

One more thing. Building `edisassm` from it's own repository using `cmake` works.

I hope you can help. Meanwhile I'll use the `edb-debugger` from Fedora's `yum` repositories.

With respect,
~ Cristian "Dr.Optix" Dinu

**Comment by eteran** [comment]
I'll look I to it. I think there is a way to clone the sub modules fully,
I'll let you know.

Also, it looks at first glance like you are building with Qt5 but I think
it should work

**Comment by eteran** [comment]
OK, I checked it out. So, the basic problem is that the latest code in edisassm has a changed API from the latest tagged version. So doing a separate checkout and symlinking is not the easiest solution.  Fortunately, there is a trivial fix :-).

When you do the git clone run the command with the `--recursive` flag and it should pull down the submodules and make sure they are set to the appropriate versions as well. For example:

    git clone --recursive git@github.com:eteran/edb-debugger.git

produces a properly building source tree here. Please re-open the issue if this does not resolve the problem.

Thanks.

**Comment by DrOptix** [comment]
Just to let you know.

At first I was not able to fully clone the repository + submodules because the submodules are setup using SSH and not HTTPs. I added a SSH key to my account and now I was able to clone the repository and the submodules are pointing to the right point in their own time line.

Another thing. I just build it using Qt5 t works without a problem.

With respect,
~ Cristian "Dr.Optix" Dinu
